web/themes/custom/cloudy/templates/field/field--field-linkedin.html.twig

```diff
@@ -1,18 +0,0 @@
-{% embed 'field.html.twig' with {
```

Comment:
@brittanysmart, shouldn't we keep these templates for the deprecated field until we have updated the data? If we delete them, the output will break from the current expected layout even if the value is actually there. That will cause the usage to look broken until Oden can get the audit complete. We can make part of his audit the removal of the data in the field so that when complete we can remove the field definitions and these templates in a separate PR. Sound good?

Reply:
AFAIK the deprecated YouTube, LinkedIn, and Nextdoor group fields are not displayed which is why I renamed the template to use our newly added fields. Group homepage templates display either a contact (uses contact item component) or Facebook, Twitter, Instagram fields.

Is there a place where the deprecated fields are currently displayed that I've overlooked?

```
{% set marquee_secondary_right %}
  {% if content.field_contact.0 %}
    {{ content.field_contact }}
  {% else %}
    {{ content.field_email }}
    {{ content.field_phone }}
    {{ content.field_facebook }}
    {{ content.field_twitter }}
    {{ content.field_instagram }}
  {% endif %}
{% endset %}
```